domain/sequence.go

```diff
@@ -0,0 +1,54 @@
+// Copyright 2025 Canonical Ltd.
+// Licensed under the AGPLv3, see LICENCE file for details.
+
+package domain
+
+import (
+	"context"
+	"database/sql"
+	
+	"github.com/canonical/sqlair"
+
+	"github.com/juju/juju/internal/errors"
+)
+
+type sequence struct {
+	Namespace string `db:"namespace"`
+	Next      uint   `db:"next_value"`
```

Comment:
I'd rather that just be called sequence or value. It's not the next value, it's the current value, as you read it from the SELECT.

Reply:
ok